.github/workflows/_graph_proxy_code.yaml

```diff
@@ -63,5 +63,6 @@ jobs:
         working-directory: graph-proxy
         run: >
           cargo test
+          --release
```

Comment:
Why do we need `--release` here?

Reply:
I found that running certain tests that make graphql queries with `cargo test` (no  `--release`) suffers the same issue as when running the graph-proxy with `cargo run serve` (no  `--release`): tokio reports a stack overflow when handling a query, and the test fails as a consequence.

I'll take another look at that stack overflow issue; revisiting the github issue I looked at before which mentioned the same issue, it looks like it was resolved early in December https://github.com/async-graphql/async-graphql/issues/1506

Reply:
Ok, I used a `.cargo/config.toml` file to specify the `RUST_MIN_STACK` env var. [Docs](https://doc.rust-lang.org/cargo/reference/config.html#configuration-format) say this will be present for any process that `cargo` runs, so this also gets rid of the stack overflow when running `cargo run serve`.

I'm not familiar with this `.cargo/config.toml` file though, so let me know if this isn't quite the right config.

(The other thing I found in rust docs for setting env vars was [build script stuff](https://doc.rust-lang.org/cargo/reference/build-scripts.html#rustc-env), though they explicitly say this should only be used for env var inspection at compile-time using `env!`).